creators/templatetags/creator_books.py

```diff
@@ -31,12 +31,17 @@ def get_creator_books(creator: Creator, perms):
     while series_len < len(series):
         series_len = len(series)
         series = series | set(Series.objects.filter(part_of_series__in=series))
+    subworks = set(Work.objects.filter(creatortowork__creator=creator))
 
-    result = []
+    item_set = set()
     for work in Publication.objects.filter(
-            Q(creatortowork__creator=creator) | Q(workinseries__part_of_series__in=series)):
+            Q(creatortowork__creator=creator) | Q(workinseries__part_of_series__in=series) | Q(workinpublication__work__in=subworks)):
```

Comment:
I expect something like `workinpublication__work__creatortowork__creator=creator` to work here as well.
It's a bit of a long string, but it does mean we let postgres do all the smart stuff :).